crates/miden-objects/src/account/code/mod.rs

```diff
@@ -260,7 +251,7 @@ impl Eq for AccountCode {}
 impl Serializable for AccountCode {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
         self.mast.write_into(target);
-        // since the number of procedures is guaranteed to be between 1 and 256, we can store the
+        // since the number of procedures is guaranteed to be between 2 and 256, we can store the
```

Comment:
Is changing `1` to `2` here correct? Could we not have accounts with just a single procedure?